docs/extensions/hooks.md

```diff
@@ -49,7 +49,9 @@ Filter hooks act on the event's payload before the event is fired. They allow yo
 Below is an example of canceling a `create` event by throwing a standard Directus exception.
 
 ```js
-import { InvalidPayloadError } from '@directus/errors';
+import { createError } from '@directus/errors';
+
+const InvalidPayloadError = createError('INVALID_PAYLOAD_ERROR', 'Something went wrong...', 500);
```

Comment:
You create `InvalidPayloadError`, but using `InvalidPayloadException`...

Reply:
Resolved in https://github.com/directus/directus/pull/19122 🙂